### DISCUSSION TITLE
Keynote - Nina da Hora

### DISCUSSION THREAD

**Discussion by pedrolacerda** [gittogethers/octogatosconf22#153] [Talks in Portuguese]
Quaisquer perguntas para a Nina, podem dropar aqui nessa discussão 😄

**Comment by bpires** [comment]
Nina, como alguém que vem das humanas/sociais aplicadas, migrando pra área de TI, tenho cada vez mais me surpreendido com vozes que trazem reflexão ética e social sobre tecnologia de dentro da indústria! 

Confesso que é algo que não esperava. Fazia engenharia mecânica há quase 15 anos e o que me fez mudar de área na época foi justamente me sentir um peixe fora d'água com tanta gente descolada da realidade... 🤐 Fico feliz de chegar agora na TI e encontrar um cenário diferente, ou pelo menos com várias iniciativas tentando mudá-lo

Mas como você vê o estado atual desse movimento por responsible tech / justiça algorítmica? Como é a recepção das empresas e desenvolvedores em geral quanto a isso? Estamos em plena transformação, ou ainda nadando contra a maré? `[Reality check]`
